app/javascript/packages/address-search/package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "@18f/identity-address-search",
-  "version": "1.0.5",
+  "version": "1.1.0",
   "type": "module",
```

Comment:
Do you think this warrants a major version change rather than a minor?

Reply:
I was thinking a major rather than minor.  I looked at your changes quickly so could be wrong. If there are breaking changes then I think better to do major.

Reply:
Resolving